models/issue.go

```diff
@@ -740,7 +743,16 @@ func AddDeletePRBranchComment(doer *User, repo *Repository, issueID int64, branc
 func (issue *Issue) ChangeContent(doer *User, content string) (err error) {
 	oldContent := issue.Content
 	issue.Content = content
-	if err = UpdateIssueCols(issue, "content"); err != nil {
+
+	regExpTasks := regexp.MustCompile(`(^\s*-\s\[[\sx]\]\s)|(\n\s*-\s\[[\sx]\]\s)`)
```

Comment:
This is duplicated code, please consider refactoring.

Also, it would be nice if we didn't have to compile the regexp every time.